content/chainguard/chainctl-usage/chainctl-images-history.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "chainctl images history"
+lead: ""
+description: "chainctl images history basics"
+type: "article"
+date: 2025-04-09T08:49:15+00:00
+lastmod: 2025-04-09T08:49:15+00:00
+draft: false
+tags: ["chainctl", "images", "history", "Product"]
+images: []
+weight: 75
+---
+
+This page demonstrates how to use the `chainctl images history` command. For a full reference of all commands with details and switches, see [chainctl Reference](/chainguard/chainctl/).
+
+At times it may be useful to examine the history of an image tag in chainctl, such as to find when it was updated and the associated digests for each update. This is done using `chainctl images history`, which is part of the [chainctl images](./chainctl-images.md) set of commands.
+
+Let's walk through a typical scenario. In our example we use an internal organization created for the Developer Enablement team to use when creating educational resources. Change our examples to use your org. You can do this by selecting from the options presented in a menu as you issue commands, as in [List chainctl images](#list-chainctl-images)
```

Comment:
> In our example we use an internal organization created for the Developer Enablement team to use when creating educational resources

Looks like we have "MyCorp" the example below. Should this be edited to reflect what we actually do in the below instructions?

Reply:
Doh! Yes. I started out using the list of orgs I have access to and decided to make it more obvious to the reader that they won't have the same org list. I'll fix this.
TY!!